### ISSUE TITLE
It is missing an icon

### ISSUE THREAD

**Issue by Zero3K** [maxrigout/Remote#8]
Please make an icon for it and assign it to the program.

**Comment by maxrigout** [comment]
yeah you're right, the gui could be also a lot prettier, the huge text box on the right isn't being used (I wanted this to be where all the log messages appear). So this is definitely not fully featured. I mainly started this project as a way to learn about winsock2 and input. I'm not sure if I'll ever add an icon.
